Harvest.Web/ClientApp/src/Fields/FieldLayers.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect, useState } from "react";
+import { LayersControl, GeoJSON, Popup, LayerGroup } from "react-leaflet";
+import { Field, Project } from "../types";
+
+interface Props {
+  project: Project;
+}
+
+export const FieldLayers = (props: Props) => {
+  const [fields, setFields] = useState<Field[]>([]);
+
+  // load fields for active projects
+  useEffect(() => {
+    const cb = async () => {
+      const startDate = new Date(props.project.start);
+      const endDate = new Date(props.project.end);
```

Comment:
Just start and end dates? Don't want to include active projects?

Reply:
@jSylvestre if you look at the C# controller it does filter by just approved projects.  They don't have to be currently active.